x-pack/plugins/alerting/server/routes/lib/error_handler.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import {
+  RequestHandler,
+  KibanaRequest,
+  KibanaResponseFactory,
+  RequestHandlerContext,
+  RouteMethod,
+} from 'kibana/server';
+
+export function handleDisabledApiKeysError<P, Q, B>(
+  handler: RequestHandler<P, Q, B>
+): RequestHandler<P, Q, B> {
+  return async (
+    context: RequestHandlerContext,
+    request: KibanaRequest<P, Q, B, RouteMethod>,
+    response: KibanaResponseFactory
+  ) => {
+    try {
+      return await handler(context, request, response);
+    } catch (e) {
+      if (isApiKeyDisabledError(e)) {
+        return response.badRequest({
+          body: new Error(
+            i18n.translate('xpack.alerting.api.error.disabledApiKeys', {
+              defaultMessage: 'Alerting relies upon API keys which appear to be are disabled',
```

Comment:
I think the word `are` isn't needed here.

cc @gchaps as I don't believe this was in the UI copy, let us know what this message should be? It shows up as a toast message when the user clicks save from the create / edit alert form.

Screenshot:
<img width="603" alt="Screen Shot 2020-03-23 at 11 26 41 AM" src="https://user-images.githubusercontent.com/3694571/77333016-33cdc000-6cf9-11ea-83bc-fc5d673d108f.png">

Reply:
Changed to "Alerting relies upon API keys which appear to be disabled"